clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -86,6 +84,11 @@ func createProfileMeasurementFactory(name, kind string) func() measurement.Measu
 }
 
 func (p *profileMeasurement) start(config *measurement.MeasurementConfig) error {
+	p.config.hosts = config.ClusterFramework.GetClusterConfig().MasterIPs
```

Comment:
Ideally, we'd allow the `hosts` to be specified via config.Params. So in  populateProfileConfig() we'd have
```
	if p.config.host, err = util.GetStringsOrDefault(config.Params, "hosts", config.ClusterFramework.GetClusterConfig().MasterIps); err != nil {
    return err
}
```

And here, after the populateProfileConfig, we'd have this check whether hosts are not empty.

But, this is fine, we currently don't set host anywhere via measurement params, so this can be improved in a separate PR